SPECS/cloud-hypervisor-cvm/fetch-private-deps

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
is this file required?

Reply:
Double checked and doesn't look like we need this. This seems to be a helper script used by LSG to fetch the sources from the package feed